src/status_im/extensions/core.cljs

```diff
@@ -474,6 +479,10 @@
                 {:permissions [:read]
                  :value       :extensions.chat.command/send-message
                  :arguments   {:params :map}}
+                'chat/send-message
+                {:permissions [:read]
+                 :value       :chat/send-message
+                 :arguments   {:to :string :value :string}}
```

Comment:
What's to? Can it be used for all chat types?

Reply:
':to' can be a user's address or a public channel